src/tools/rest-frontend/resources/structure.json

```diff
@@ -0,0 +1,1896 @@
+[
+    {
```

Comment:
Does the json parser being used here allow comments? If not, can you add some "info" or similar nodes and give some explanations? (not about what the types are, they are really good described in the README.md, but more specific decisions/details)